sunpy/conftest.py

```diff
@@ -36,3 +43,14 @@ def pytest_runtest_setup(item):
         if 'online' in item.keywords and not is_online():
             msg = 'skipping test {0} (reason: client seems to be offline)'
             pytest.skip(msg.format(item.name))
+
+def pytest_unconfigure(config):
+    #Check if additions have been made to the hash library
+    if len(hash.hash_library) > hash_library_original_len:
+        #Write the new hash library in JSON
+        tempdir = tempfile.mkdtemp()
+        hashfile = os.path.join(tempdir, hash.HASH_LIBRARY_NAME)
+        with open(hashfile, 'wb') as outfile:
+            json.dump(hash.hash_library, outfile, sort_keys=True, indent=4, separators=(',', ': '))
+        print "The hash library has expanded and should be copied to sunpy/tests/"
+        print "  " + hashfile
```

Comment:
we can, and if people want to do that it's fine, it is however, one of the easier things to automate for forwards compatibility.